examples/sample-app/README.md

```diff
@@ -121,6 +121,8 @@ This section covers how to perform all the steps of building, deploying, and upd
 
     Note: sudo is required so the kubernetes proxy can manipulate iptables rules to expose service ports.
 
+    Note: when using vagrant synced folder it is advised to use different directory than the one in synced folder. This can be achieved by passing `--volume-dir=/absolute/path` to `openshift start` command.
```

Comment:
"it is advices to use a different directory for volume storage rather than using the synced folder."